operator/Makefile

```
@@ -434,6 +434,7 @@ bundle-post-process: test-bundle-helpers operator-sdk ## Post-process CSV file t
 	mkdir -p build/
 	rm -rf build/bundle
 	cp -a bundle build/
+	cp ../config-controller/config/crd/bases/*.yaml build/bundle/manifests/ ;\
```

```suggestion
	cp ../config-controller/config/crd/bases/*.yaml build/bundle/manifests/
```

This seems independent of the commands below.

---

cadbc63